docs/windows.rst

```diff
@@ -104,7 +104,7 @@ You can also build a native Windows version using Autotools.
 
 If you use Autotools you can also do the Units testing with `make units`. Some tests fail, that needs to be investigated.
 
-Sometimes anti-virus product can slowdown the build and test process significantly, especially when ``./configure`` is running and during the Units tests. In that case it could help to temporary disable them. But be aware of the risks when you disable your anti-virus product.
+Sometimes anti-virus products can slowdown the build and test process significantly, especially when ``./configure`` is running and during the Units tests. In that case it could help to temporary disable them. But be aware of the risks when you disable your anti-virus product.
```

Comment:
Could begin "Some anti-virus software slows down the build...".

Last sentence could use "software" instead of "product".